pkg/resource/obtenantbackuppolicy_manager.go

```diff
@@ -393,15 +394,15 @@ func (m *ObTenantBackupPolicyManager) getOBTenantCR() (*v1alpha1.OBTenant, error
 			Resource: "obtenantbackuppolicies",
 		}, m.BackupPolicy.Spec.TenantCRName)
 	}
-	tenantName := m.BackupPolicy.Spec.TenantCRName
```

Comment:
Modify the name of variable to alert me to distinguish tenant in database and tenant resource in k8s!